orcoursetrion/lib/github.py

```diff
@@ -67,10 +72,37 @@ def _get_all(self, url):
             ):
                 response = self.session.get(response.links['next']['url'])
                 results += response.json()
-        if response.status_code != 200:
+        if response.status_code not in [200, 404]:
             raise GitHubUnknownError(response.text)
         return results
 
+    def _get_repo(self, org, repo):
+        """Either return the repo dictionary, or None if it doesn't exists.
+
+        Args:
+            org (str): Organization the repo lives in.
+            repo (str): The name of the repo.
+        Raises:
+            requests.exceptions.RequestException
+            GitHubUnknownError
```

Comment:
Right, you can see in the [api docs](http://orcoursetrion.readthedocs.org/en/latest/api/#orcoursetrion.lib.GitHub) that it figures it out and links to it.  Though there is a case to be made to put the absolute path to the exception there.  Maybe add that as an issue?